src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -982,6 +985,12 @@ def aks_create(cmd,     # pylint: disable=too-many-locals,too-many-statements,to
         mc.api_server_access_profile = ManagedClusterAPIServerAccessProfile(
             enable_private_cluster=True
         )
+    
+    if paid_high_availability:
+        mc.sku = ManagedClusterSKU(
+            name="Basic",
+            tier="Paid"
+        )
```

Comment:
Default to free tier?

Reply:
without setting, be default is "Free", it will be set at server.